astore/client/auth/auth.go

```diff
@@ -98,10 +104,39 @@ func (c *Client) Login(username, domain string, o LoginOptions) (string, error)
 		return "", fmt.Errorf("server returned invalid nonce, please try again - %s", err)
 	}
 
-	decrypted, ok := box.Open(nil, []byte(tres.Token), nonce.ToByte(), servPub.ToByte(), priv)
+	decrypted, ok := box.Open(nil, tres.Token, nonce.ToByte(), servPub.ToByte(), priv)
 	if !ok {
 		return "", fmt.Errorf("could not decrypt returned token")
 	}
 
-	return string(decrypted), nil
+	caPublicKey, _, _, _, err := ssh.ParseAuthorizedKey(tres.Capublickey)
```

Comment:
very important, just forgot about this: if I cut an enkit version today, login will break, as the server will not return a `Capbulickey`. Please make this code conditional, something like:

```
if len(tres.Capublickey) == 0 || len(tres.Cert) == 0 {
  ...
  return immediately;
}
```

This will also allow to do rollbacks cleanly in case we need to.